blog/5-must-have-ai-tools.md

```
@@ -0,0 +1,335 @@
+title: "5 Must-Have Command Line AI Tools"
+date: 2024-06-20
+tags: ['ai', 'command-line', 'automation', 'productivity']
+![](_media/984983_image0.jpg)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)